ui/packages/statement/src/components/StatementSummaryTable.tsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react'
 import { Table } from 'antd'
 import moment from 'moment'
-import { StatementDetailInfo } from './statement-types'
 import { useTranslation } from 'react-i18next'
-import SyntaxHighlighter from 'react-syntax-highlighter'
-import { atomOneLight } from 'react-syntax-highlighter/dist/esm/styles/hljs'
 import sqlFormatter from 'sql-formatter-plus'
+import { Light as SyntaxHighlighter } from 'react-syntax-highlighter'
+import sql from 'react-syntax-highlighter/dist/esm/languages/hljs/sql'
+import { atomOneLight } from 'react-syntax-highlighter/dist/esm/styles/hljs'
+import { StatementDetailInfo } from './statement-types'
+
+SyntaxHighlighter.registerLanguage('sql', sql)
```

Comment:
maybe we need to do it in the entry file to avoid registering multiple times when it is used in somewhere else in future?

Reply:
yep, I thought that, now we only have this one place need it, if we have mulitple places need it, we need to extract.